main.go

```diff
@@ -192,37 +194,47 @@ func (o *Options) runCLIReportMode() error {
 		DisplayDataConfig:           o.toDisplayDataConfig(),
 	}
 
-	testReport := analyzer.PrepareTestReport(o.ToTestGridDashboardCoordinates()[0], o.getVariantManager(), o.getBugCache())
+	testReport := analyzer.PrepareTestReport(o.ToTestGridDashboardCoordinates()[0], o.getSynthenticTestManager(), o.getVariantManager(), o.getBugCache())
 	enc := json.NewEncoder(os.Stdout)
 	enc.Encode(testReport)
 	return nil
 }
 
-func (o *Options) hasOpenshiftRelease() bool {
+func (o *Options) hasOpenshiftDashboard() bool {
```

Comment:
you may want to preemptively start referring to this as the OCP dashboard, since there is also an OKD dashboard that might get pulled in at some point.